discord/api/api.go

```diff
@@ -0,0 +1,165 @@
+package api
+
+// Api is a facade between the discord package and the backend
+
+import (
+	"log"
+
+	"github.com/zond/diplicity/game"
+)
+
+type Province struct {
+	Name     string
+	Key      string
+	UnitType string
+}
+
+type OrderType struct {
+	Name string
+	Key  string
+}
+
+type Api struct {
+}
+
+func (a *Api) SourceProvinces(userId, channelId string) ([]Province, error) {
```

Comment:
Note, the methods here which get the provinces/order types are just returning static data. Will get correct data in a later PR